server/portal/apps/search/tasks.py

```diff
@@ -52,7 +52,7 @@ def index_community_data(self, reindex=False):
     for sys in settings.PORTAL_DATAFILES_STORAGE_SYSTEMS:
         if sys['api'] == 'tapis' and sys['scheme'] != 'private':
```

Comment:
We should update this as well to only index `public` or `community` schemes, so as to ignore both `private` and `projects` schemes.
```suggestion
        if sys['api'] == 'tapis' and sys['scheme'] in ['community', 'public']:
```